metadata/launchdarkly_dotnet-core.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
You can delete this file (or not, it will be fetched at crawl time and overwritten)

Reply:
Although, it looks like this metadata file has more info than the one checked into the repo: https://github.com/launchdarkly/dotnet-core/blob/main/.sdk_metadata.json

The repo's file should be updated to contain this content

Reply:
There is a PR in the dotnet repo that will update it.  Is order of operations critical?

Reply:
Deleted